charts/prometheus-adapter/Chart.yaml

```diff
@@ -1,6 +1,6 @@
 apiVersion: v1
 name: prometheus-adapter
-version: 2.16.0
+version: 2.17.0
```

Comment:
Changing labels will break updates. So this would be a major version change.
Please add upgrading section to readme.

Reply:
@monotek so I must change the chart version to `v3.0.0` ?